### PR TITLE
Fixes most of mark's symptoms

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
Makes new viro symptoms viable
## Why It's Good For The Game
New symptoms were balanced very poorly, with their timers being abysmal to the degree that you would never even see a clown mask spawn when you met the threshold with pierrots
## Changelog
:cl:
tweak: increased all of the new symptom's speeds
tweak: also changed revitiligo's stats so it's no longer a duplicate of coughing.
/:cl:

NOTE: Ducatopod's slip still doesnt actually slip, and it still has no trigger to apply the clown shoes. I still dunno shit about code, so mark'll need to fix this

OTHER NOTE: beesease bees are currently screwed up too. negative beesease spawns random reagent bees, and honey beesease spawns normal toxin bees. I think you need to switch them around- in code, the bees marked toxin bees are random bees

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/beesymptom.dm [hunk 1]
```diff
@@ -30,24 +30,24 @@
 		if(2)
 			if(prob(2))
 				to_chat(M, "<span class='notice'>You taste honey in your mouth.</span>")
-		if(3)
-			if(prob(10))
+		if(3, 4, 5)
```
Comment (MarkSuckerberg) [inline code/datums/diseases/advance/symptoms/beesymptom.dm]: As ike told me, we can't do this with switches. We can't call the stages more than once, or it'll just get stuck on this step and won't ever process the other ones.

Comment (zeskorion) [reply code/datums/diseases/advance/symptoms/beesymptom.dm]: Ah. Well, when you take care of the other issue, please find some way to make this proc at later stages too

File: code/datums/diseases/advance/symptoms/lubefeet.dm [hunk 2]
```diff
@@ -31,14 +31,14 @@
 			if(prob(15))
 				to_chat(M, "<span class='notice'>Your feet begin to sweat profusely...</span>")
 		if(3, 4, 5)
-			if(prob(10))
+			if(prob(100))
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/lubefeet.dm]: > if(prob(100))

nice meme

File: code/datums/diseases/advance/symptoms/lubefeet.dm [hunk 3]
```diff
@@ -31,14 +31,14 @@
...
 				to_chat(M, "<span class='danger'>You slip from the lube from your feet!</span>")
 				M.slip()
 			if(A.stage == 4 || A.stage == 5)
 				if(morelube)
-					makelube(M, 25)
+					makelube(M, 100)
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/lubefeet.dm]: >another if(prob(100))

nice meme x2

File: code/datums/diseases/advance/symptoms/pierrot.dm [hunk 4]
```diff
@@ -32,24 +32,24 @@
...
 			if(clumsy)
 				if(!HAS_TRAIT(M, TRAIT_CLUMSY))
 					to_chat(M, "<span class='notice'>You feel dumber.</span>")
 					ADD_TRAIT(M, TRAIT_CLUMSY, DISEASE_TRAIT)
-			if(prob(10))
+			if(prob(30))
 				M.say( pick( list("HONK!", "Honk!", "Honk.", "Honk?", "Honk!!", "Honk?!", "Honk...") ))
 			if(A.stage == 5)
-				if(prob(1) && prob(50) && clownmask)
+				if(prob(100) && prob(100) && clownmask)
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/pierrot.dm]: going for a hat trick I see

File: code/datums/diseases/advance/symptoms/wizarditis.dm [hunk 5]
```diff
@@ -28,27 +28,27 @@
...
+			if(prob(30) && prob(50))
 				M.say(pick("NEC CANTIO!","AULIE OXIN FIERA!", "STI KALY!", "TARCOL MINTI ZHERI!"))
-			if(prob(1) && prob(50))
+			if(prob(30) && prob(50))
 				to_chat(M, "<span class='danger'>You feel [pick("the magic bubbling in your veins","that this location gives you a +1 to INT","an urge to summon familiar")].</span>")
 
 		if(4)
 
-			if(prob(1))
+			if(prob(100))
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/wizarditis.dm]: the ride never ends

File: code/datums/diseases/advance/symptoms/wizarditis.dm [hunk 6]
```diff
@@ -28,27 +28,27 @@
...
 				M.say(pick("NEC CANTIO!","AULIE OXIN FIERA!","STI KALY!","EI NATH!"))
 				return
-			if(prob(1) && robes)
+			if(prob(100) && robes)
```
Comment (ike709) [inline code/datums/diseases/advance/symptoms/wizarditis.dm]: insert funny joke here

**Review by ike709 — CHANGES_REQUESTED** [review]
No `prob(100)`. If you really want it to be guaranteed, just remove the prob check.

**Comment by zeskorion** [comment]
> No `prob(100)`. If you really want it to be guaranteed, just remove the prob check.

aight, fixed it. As ya know, i dont know shit about code, so im skittish about removing anything. this should fix it

**Pull request merged**